src/components/Project/Project.tsx

```diff
@@ -24,39 +24,113 @@ export default function Project({ project }: Props): JSX.Element {
 
   return (
     <Root>
-      <Header>
-        <EditableTitle value={project.name} onBlur={handleBlur} />
-
-        <StarButton data-testid="starButton">
-          <Star />
-        </StarButton>
-      </Header>
-
-      <Panels>
-        <DnDColumnList projectColumns={project.columns} />
-        <AnotherListButton title="Add another list" icon={<Add />} textLeft />
-      </Panels>
+      <Container>
+        <Header>
+          <HeaderLeft>
+            <Button title="Board" icon={<TableChart />} data-testid="boardButton" />
+
+            <EditableTitle value={project.name} onBlur={handleBlur} />
+
+            <StarButton data-testid="starButton">
+              <StarBorder />
+            </StarButton>
+
+            <Button title="test" data-testid="testButton" />
+
+            <Button title="Workspace visible" icon={<People />} data-testid="workSpaceButton" />
+
+            <ShareButton title="Share" icon={<GroupAdd />} data-testid="shareButton" />
```

Comment:
[personal preference] - setting next line on every single group JSX elements looks too spacious to me😄  It makes me feel uncomfortable reading it through personally.